Deprecate app (#49)
vtex-apps/product-kit: docs/README.md

# VTEX Product Kit ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)

> :warning: **The product-kit app has been deprecated** (support is still given).

Reviewer: ```suggestion
⚠️ **The Product Kit app has been deprecated.** Despite this, support for it is still granted.
```
